book/tutorials/nb-to-package/intro.md

```diff
@@ -0,0 +1,270 @@
+# Packaging notebook introduction and demo
+
+This tutorial demonstrates how powerful packaging your Python code
+can be ... even for yourself.
+
+Before we get ahead of ourselves. Let's think about the **WHY**.
+You might ask yourself:
+
+- why would I want to package my code?
+- can't I just pass around notebooks?
+- what's the point of all this?
+- what is the meaning of life?
+
+... well, maybe not the last one. But the others are valid questions.
+
+## What is a Python package?
+
+A "package" or officially called a "distribution package" is simply
+a directory of Python modules and sub-packages with an `__init__.py` file
+that allows for the modular organization of code.
+It can be be installed and distributed for use by others.
+
+![Bedroom](https://images.unsplash.com/photo-1630699293259-0b6c08606c62?q=80&w=3870&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D)
+
+Let's think about this in a simpler term. We can think of a
+package like a room in a house. Say the "bedroom" is a package.
+Inside the bedroom, you have a bed, a nightstand, and a closet.
+
+- The bed and nightstand are like *modules* in a package.
+- The closet is like a *sub-package* in a package, since there might be other
+furniture pieces in a closet like a dresser; some closets can even be a whole room!
+
+### Module
+
+A module is a single file of Python code that can be imported into Python
+scripts or other modules,
+also known officially as an "import package".
```

Comment:
```suggestion
scripts or other modules.
```

Reply:
(I find this really confusing as it seems like modules and libraries are "packages" so almost everything could be called a package 😅) I suggest being more opinionated for clarity about the hierarchy... Would it be too bold to claim that a "package" is strictly a "library" that can be installed with pip :) ?

Reply:
"modules" are not packages.. seems like that's not clear 😛 "libraries" are "packages", I've called this part out later, see https://github.com/ICESAT-2HackWeek/website-2024/pull/14/files#r1717208218. I want to highlight this fine differences hence the bedroom analogy. I'll think about how to make this more clear, but if you have any ideas, please let me know. I think it's an important difference to know.

Reply:
I read this sentence as  `a "module" also known officially as an "import package"` I just recommend removing that last bit of the sentence